shopping_list_frontend/lib/screens/home.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:shopping_list_frontend/cubits/user/user_cubit.dart';
+import 'package:shopping_list_frontend/services/lifecycle/navigator.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final userCubit = context.read<UserCubit>();
+    final appNavigator = AppNavigator(context);
+
+    final userState = context.read<UserCubit>().state;
```

Comment:
Better to use `BlocBuilder` or `BlocSelector` for that staff
It's more stable and responsible as api